src/structures/GuildMember.js

```diff
@@ -83,6 +83,15 @@ class GuildMember extends Base {
     }
 
     if ('nick' in data) this.nickname = data.nick;
+    if ('avatar' in data) {
+      /**
+       * The ID of the member's guild avatar
+       * @type {?string}
+       */
+      this.avatar = data.avatar;
+    } else if (typeof this.avatar !== 'string') {
+      this.avatar = null;
+    }
```

Comment:
```suggestion
    if ('avatar' in data) {
      /**
       * The ID of the member's guild avatar
       * @type {?string}
       */
      this.avatar = data.avatar;
    }
```
If `avatar` field is present in the response, then it will be either `null` or `string`. So, the `else if` check seems redundant here. Also, why were you checking for `this.avatar` in it anyway?

Reply:
I think the goal is to initialize the property with null if the member has no guild avatar.

Reply:
hmm, I see. Wouldn't it be a good idea to have it initialized to `null` in constructor then?

Reply:
I used the same checks that were used in User#avatar so that they’re consistent with each other but I can change that if you think it’s necessary